captura/views.py

```diff
@@ -202,3 +205,16 @@ def capturista_dashboard(request):
             capturista=Capturista.objects.get(user=request.user))
     return render(request, 'captura/dashboard_capturista.html',
                   {'estudios': estudios, 'Estudio': Estudio})
+
+
+@login_required
+@user_passes_test(is_capturista)
+def create_estudio(request):
+    """ This view creates the family, and estudio entities that are
+    required for the creation and fullfillment of every piece of functionality
+    in this app.
+    """
+    if request.method == 'POST':
+        familia = Familia.objects.create()
+        Estudio.objects.create(capturista=request.user.capturista, familia=familia)
+        return redirect(reverse('home'))
```

Comment:
I guess this should be changed to redirect directly to the section where we start to fill the study, once we have that, right?